test/src/client_utils.mocks.dart

```diff
@@ -0,0 +1,91 @@
+import 'dart:async' as _i3;
```

Comment:
I am surprised this file does not have GENERATED FILE warning at the top.

Reply:
Created https://github.com/dart-lang/mockito/issues/322